modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-1_{context}"]
+= Release notes for {gitops-title} 1.1
+
+[id="gitops-features-1-1_{context}"]
+== GitOps features
+
+.GitOps features
+[cols="1,1",options="header"]
+|===
+| {gitops-title} Product | Release
+| Argo CD
+| General availability (GA)
+| Argo CD ApplicationSet
+| Tech Preview (TP)
```

Comment:
Technology preview in all instances.

Reply:
We can add a key like the core OCP RN does: In the table below, features are marked with the following statuses:

TP: Technology Preview

GA: General Availability

Reply:
Done